src/main/java/org/humancellatlas/ingest/state/MetadataDocumentEventHandler.java

```diff
@@ -21,6 +22,8 @@ public void metadataDocumentAfterCreate(MetadataDocument document) {
 
     public void handleMetadataDocumentCreate(MetadataDocument document) {
         messageRouter.routeValidationMessageFor(document);
-        messageRouter.routeStateTrackingUpdateMessageFor(document);
+        if (document.getSubmissionEnvelope() != null) {
```

Comment:
The intent might be clearer if we changed this to:

```
if(document.getEntityType != PROJECT)
```

Reply:
I am not sure if we're still making it possible to create a project from an empty submission envelope. If we are then the current condition will support both scenarios: Project first or Submission first.

Reply:
Added it as OR condition to only allow for projects